### PR TITLE
Create 0000-01-02-susan246.md

### PR DESCRIPTION
**Pull request by aayushi246**
created a branch, created a file and made a commit, and opened a pull request

### CODE REVIEW

File: _posts/0000-01-02-susan246.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (susan246) [inline _posts/0000-01-02-susan246.md:5]: meme

Comment (susan246) [inline _posts/0000-01-02-susan246.md:5]: meme

Comment (aayushi246) [inline _posts/0000-01-02-susan246.md:5]: n

**Review by aayushi246 — COMMENTED** [review]
done

**Pull request closed without merging**